### ISSUE TITLE
Add additional fields for relationships [2]

### ISSUE THREAD

**Issue by kconn** [public-accountability/littlesis-browser-addon#19]
priorities:
- start date and end date for all
- amount for some
- for position, is_board (yes no unknown)

**Comment by lizstarin** [comment]
@kconn It's easy enough to add fields—what do we want to do with them?

**Comment by kconn** [comment]
more notes on this:

- is_board should only appear when a "position" relationship is selected. is_board should allow for yes, no, or unknown
- start date and end date can be applied to any relationship. they will need to be validated for accepted date formats, ideally using whatever validation other forms use. 
- amount: though amounts are stored in the same field in the db, the label changes based on the type of relationship (eg, for position it is labeled as "compensation")...for now, though, I don't think we should bother with changing the label unless it is easy.
Everything labeled with @@money here is an amount field, I believe, and those relationships. https://github.com/public-accountability/littlesis-rails/blob/c3d2e25289ad3d2068355c976702dd056392857c/lib/relationship_details.rb

As for layout/design, my suggestions:
- get of the swap button; make amount field appear on same line as is_current
- put start date and end date on same line below it
- make is_board appear below dates when position is selected, 
- presumably we don't need "set current tab" button any more

**Comment by lizstarin** [comment]
Are there any types of relationships that shouldn't show "amount"--family, for instance?
  @kconn

**Comment by kconn** [comment]
Yes – according to [this](https://github.com/public-accountability/littlesis-rails/blob/c3d2e25289ad3d2068355c976702dd056392857c/lib/relationship_details.rb), generic, family, social, professional, ownership, hierarchy don't have amount fields (hence no @@ money). @aepyornis , is there a better way of telling that? also, i'm not sure why ownership doesn't have an amount field, it sort of feels like value would be a good field to have there. I thought we used to have that, actually.

**Comment by lizstarin** [comment]
![new-layout-1](https://user-images.githubusercontent.com/3151071/34849838-bd8fe41a-f6f1-11e7-87cc-35d5f18e736c.png)
![new-layout-2](https://user-images.githubusercontent.com/3151071/34849839-bda0d54a-f6f1-11e7-88eb-12c7952495f0.png)
![new-layout-3](https://user-images.githubusercontent.com/3151071/34849840-bdb33442-f6f1-11e7-9d4b-6d6683914b41.png)
